### DISCUSSION TITLE
Could not Installed "spatie/flysystem-dropbox" package on Laravel 8.35.1 version reason of some dependecies.

### DISCUSSION THREAD

**Discussion by dspias** [spatie/flysystem-dropbox#69] [Q&A] [answered]
When I try to install it its show this error

Problem 1

    ```
- spatie/flysystem-dropbox[2.0.0, ..., v2.x-dev] require league/flysystem ^2.0.4 -> found league/flysystem[2.0.4, 2.x-dev] but the package is fixed to 1.1.3 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires spatie/flysystem-dropbox ^2.0 -> satisfiable by spatie/flysystem-dropbox[2.0.0, 2.0.1, 2.0.2, v2.x-dev].
```

then I try to update "league/flysystem", then it was show error like below.

Problem 1
    ```
- laravel/framework is locked to version v8.35.1 and an update of this package was not requested.
    - laravel/framework v8.35.1 requires league/flysystem ^1.1 -> found league/flysystem[1.1.0, ..., 1.x-dev] but it conflicts with your root composer.json require (^2.0).
```

  Problem 2

   ```
 - spatie/laravel-medialibrary is locked to version 9.0.0 and an update of this package was not requested.
    - spatie/laravel-medialibrary 9.0.0 requires league/flysystem ^1.0.64 -> found league/flysystem[1.0.64, ..., 1.x-dev] but it conflicts with your root composer.json require (^2.0).
```

  Problem 3

```
    - laravel/framework v8.35.1 requires league/flysystem ^1.1 -> found league/flysystem[1.1.0, ..., 1.x-dev] but it conflicts with your root composer.json require (^2.0).
    - laravel/sail v1.4.10 requires illuminate/console ^8.0|^9.0 -> satisfiable by laravel/framework[v8.35.1].
    - laravel/sail is locked to version v1.4.10 and an update of this package was not requested.
```

**Comment by freekmurze** [comment, marked as answer]
L9 will add compatibilty with flysystem 2

**Reply by dspias** [reply]
Sorry, Didn't get your point. Can you please brief more..?

**Reply by dspias** [reply]
> L9 will add compatibilty with flysystem 2

What is L9..?
@freekmurze

**Reply by freekmurze** [reply]
Laravel 9

**Reply by dspias** [reply]
Thank you

**Comment by samuelhgf** [comment]
@pd-sperrow I answered you in LaraCast 
https://laracasts.com/discuss/channels/laravel/could-not-installed-spatieflysystem-dropbox-package-on-laravel-8351-version-reason-of-some-dependecies

**Comment by Bazzly** [comment]
this error on laravel 8....
Problem 1
    - spatie/flysystem-dropbox[2.0.0, ..., v2.x-dev] require league/flysystem ^2.0.4 -> found league/flysystem[2.0.4, ..., 2.x-dev] but the package is fixed to 1.1.5 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires spatie/flysystem-dropbox ^2.0 -> satisfiable by spatie/flysystem-dropbox[2.0.0, ..., v2.x-dev].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

`Any way out ???`

**Reply by Bazzly** [reply]
owner said......Solution for laravel 9 below.

Thanks, @samuelhgf - running composer require spatie/flysystem-dropbox "^1.0.0" did the trick!
